packages/examples/server/src/server.ts

```diff
@@ -3,17 +3,12 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 import { WebSocketServer } from 'ws';
-import * as http from 'http';
-import { fileURLToPath, URL } from 'url';
-import * as net from 'net';
+import { IncomingMessage } from 'http';
+import { URL } from 'url';
+import { Socket } from 'net';
 import express from 'express';
-import * as rpc from 'vscode-ws-jsonrpc';
-import { launch } from './json-server-launcher.js';
-
-// solve: __dirname is not defined in ES module scope
-import path from 'path';
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = path.dirname(__filename);
+import { IWebSocket } from 'vscode-ws-jsonrpc';
+import { getLocalDirectory, launch } from './json-server-launcher.js';
```

Comment:
I'm not sure `json-server-launcher` is the right place to put the `getLocalDirectory` method

Reply:
It is used there as well. Any good idea where to put it? New file and both reference that one?

Reply:
That's probably what I would have done, sbut I don't know how to name it :grin:

Reply:
Ok, I go for " iDontKnowHowToNameIt.ts". 😆 No, seriously, I do that and come up with a hopefully fitting name.